### PR TITLE
APPSEC-1083: S6382 Fixed incorrect ARM examples

### PR DESCRIPTION
**Pull request by daniel-teuchert-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S6382/azureresourcemanager/rule.adoc [hunk 1]
```diff
@@ -290,7 +294,7 @@ resource example 'Microsoft.SignalRService/webPubSub@2020-07-01-preview' = {
   name: 'example'
   properties: {
     tls: {
-      clientCertEnabled: true
+      clientCertEnabled: true // Compliant
```
Comment (loris-s-sonarsource) [inline rules/S6382/azureresourcemanager/rule.adoc:297]:
```suggestion
      clientCertEnabled: true
```

no, we do not add the "compliant" comments haha

File: rules/S6382/azureresourcemanager/rule.adoc [hunk 2]
```diff
@@ -320,7 +324,7 @@ resource example 'Microsoft.SignalRService/webPubSub@2020-07-01-preview' = {
 resource example 'Microsoft.Web/sites@2015-08-01' = {
   name: 'example'
   properties: {
-    clientCertEnabled: true
+    clientCertEnabled: true // Compliant
```
Comment (loris-s-sonarsource) [inline rules/S6382/azureresourcemanager/rule.adoc:327]:
```suggestion
    clientCertEnabled: true
```

File: rules/S6382/azureresourcemanager/rule.adoc [hunk 3]
```diff
@@ -420,7 +428,7 @@ resource example 'Microsoft.DataFactory/factories/linkedservices@2018-06-01' = {
   properties: {
     type: 'Web'
     typeProperties: {
-      authenticationType: 'ClientCertificate'
+      authenticationType: 'ClientCertificate' // Compliant
```
Comment (loris-s-sonarsource) [inline rules/S6382/azureresourcemanager/rule.adoc:431]:
```suggestion
      authenticationType: 'ClientCertificate'
```

File: rules/S6382/azureresourcemanager/rule.adoc [hunk 4]
```diff
@@ -455,7 +463,7 @@ Where a list of permitted client certificates must be provided:
 resource example 'Microsoft.DocumentDB/cassandraClusters@2021-10-15' = {
   name: 'example'
   properties: {
-    clientCertificates: [
+    clientCertificates: [ // Compliant
```
Comment (loris-s-sonarsource) [inline rules/S6382/azureresourcemanager/rule.adoc:466]:
```suggestion
    clientCertificates: [
```

File: rules/S6382/azureresourcemanager/rule.adoc [hunk 5]
```diff
@@ -497,7 +505,7 @@ resource example 'Microsoft.ContainerRegistry/registries/tokens@2022-12-01' = {
   name: 'example'
   properties: {
     credentials: {
-      certificates: [ 
+      certificates: [ // Compliant
```
Comment (loris-s-sonarsource) [inline rules/S6382/azureresourcemanager/rule.adoc:508]:
```suggestion
      certificates: [
```

**Review by loris-s-sonarsource — CHANGES_REQUESTED** [review]
Things look good, however please remove all the "Compliant" comments 👍

**Review by loris-s-sonarsource — CHANGES_REQUESTED** [review]
I didn't notice them, but sometimes it is written "Sensitive", even in the code samples. Sorry for not noticing earlier, please remove them

**Comment by daniel-teuchert-sonarsource** [comment]
> I didn't notice them, but sometimes it is written "Sensitive", even in the code samples. Sorry for not noticing earlier, please remove them

What do you mean? I think this is compliant with the general guidelines to mark sensitive code in hotspots with a comment.
See: https://xtranet-sonarsource.atlassian.net/wiki/spaces/LANG/pages/2808479903/General+Guidelines#Highlight-the-sensitive-line-in-non-compliant-samples-(Readability%2C-Consistency)

**Comment by loris-s-sonarsource** [comment]
@daniel-teuchert-sonarsource I meant in the compliant code samples. Sensitive should be added in the noncompliant ones, but not the compliant ones

**Comment by daniel-teuchert-sonarsource** [comment]
> @daniel-teuchert-sonarsource I meant in the compliant code samples. Sensitive should be added in the noncompliant ones, but not the compliant ones

oops good catch!

**Review by loris-s-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**